File: Assets/Scripts/Spells/Priest/HealingWord.cs
using UnityEngine;

public class HealingWord : Spell {
    public override void Cast(Character caster, Character target) {
        CombatManager.instance.Damage(target, damage);
        caster.currentMana -= manaCost;
    }

    public override void UpdateDescription() {
        description = $"Heals the target for {damage} health.";
    }
}

File: Assets/Scripts/Lua/Spells/Priest/HealingWord.lua
function Cast(caster, target)
    caster.currentMana = caster.currentMana - spell.manaCost
    combatManager.Heal(target, spell.damage + caster.intelligence, caster)
    caster.Sync()
end

function UpdateDescription()
    spell.description = "Heals the target for", spell.damage, "health.";
end